adsingestp/parsers/jats.py

```diff
@@ -969,7 +975,8 @@ def _parse_permissions(self):
                 license_text = p.find("license-p")
                 if license_text:
                     self.base_metadata.setdefault("openAccess", {}).setdefault(
-                        "license", license_text.get_text()
+                        "license",
```

Comment:
unrelated to pubdate, but I fixed this here because of inconsistencies with handling license strings depending on whether the xml was pretty-printed or not.